sections/ads.txt

```diff
@@ -1514,6 +1518,9 @@ ugytudjuk.hu#?#.row:-abp-has(.mainTopBnr)
 ||adfair.hu^
 ||adfree-stat.p3g.hu^
 ||adk2x.com^
+! https://github.com/hufilter/hufilter-dev/issues/314
+||adskeeper.co.uk^
```

Comment:
`||adskeeper.com^` and `||adskeeper.co.uk^` exist in Easylist.

Reply:
Removed those 2 rules.